simson/plastics/plastics_definition.py

```diff
@@ -113,25 +144,44 @@ def get_definition(cfg: GeneralCfg):
-        fd.ParameterDefinition(name="chemical_recycling_rate", dim_letters=("t", "m")),
-        fd.ParameterDefinition(name="solvent_recycling_rate", dim_letters=("t", "m")),
-        fd.ParameterDefinition(name="incineration_rate", dim_letters=("t", "m")),
-        fd.ParameterDefinition(name="uncontrolled_losses_rate", dim_letters=("t", "m")),
+        fd.ParameterDefinition(name="collection_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="mechanical_recycling_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="chemical_recycling_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="solvent_recycling_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="incineration_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="landfill_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="wasteimport_rate", dim_letters=("t", "r", "g")),
+        fd.ParameterDefinition(name="wasteexport_rate", dim_letters=("t", "r", "g")),
+        # fd.ParameterDefinition(name="finalimport_rate", dim_letters=("t","r")),
+        # fd.ParameterDefinition(name="finalexport_rate", dim_letters=("t","r")),
         # virgin production rates
-        fd.ParameterDefinition(name="bio_production_rate", dim_letters=("t", "m")),
-        fd.ParameterDefinition(name="daccu_production_rate", dim_letters=("t", "m")),
+        fd.ParameterDefinition(name="bio_production_rate", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="daccu_production_rate", dim_letters=("t", "r", "m")),
         # recycling losses
-        fd.ParameterDefinition(name="mechanical_recycling_yield", dim_letters=("t", "m")),
-        fd.ParameterDefinition(name="reclmech_loss_uncontrolled_rate", dim_letters=("t", "m")),
+        fd.ParameterDefinition(name="mechanical_recycling_yield", dim_letters=("t", "r", "m")),
+        fd.ParameterDefinition(name="reclmech_loss_uncontrolled_rate", dim_letters=("t", "r", "m")),
         # other
-        fd.ParameterDefinition(name="material_shares_in_goods", dim_letters=("m", "g")),
+        fd.ParameterDefinition(name="material_shares_in_goods", dim_letters=("r", "m", "g")),
         fd.ParameterDefinition(name="emission_capture_rate", dim_letters=("t",)),
         fd.ParameterDefinition(name="carbon_content_materials", dim_letters=("e", "m")),
         # for in-use stock
+        fd.ParameterDefinition(name="wasteimporttotal", dim_letters=("t", "g")),
+        fd.ParameterDefinition(name="finalimporttotal", dim_letters=("t",)),
         fd.ParameterDefinition(name="production", dim_letters=("h", "r", "g")),
-        fd.ParameterDefinition(name="lifetime_mean", dim_letters=("g",)),
-        fd.ParameterDefinition(name="lifetime_std", dim_letters=("g",)),
+        fd.ParameterDefinition(
+            name="lifetime_mean",
+            dim_letters=(
+                "r",
+                "g",
+            ),
```

Comment:
```suggestion
                "g"
            )
```
This will cause the formatter to make it back into a single line.

Reply:
done